docs/src/content/sections/faq.md

```diff
@@ -4,11 +4,17 @@ title: "Frequently Asked <em>Questions</em>"
 list:
   - enable: true
     question: "What is included in the trial plan?"
-    answer: "Trial plan include everything available for the Teams plan."
+    answer: "The Trial plan includes access to all 30+ RocketSim Pro & Teams features."
+  - enable: true
+    question: "Can I invite my whole team during the Teams trial?"
+    answer: "Yes, you can invite your entire team. Simply specify your team size when starting the trial, and you'll receive the corresponding number of seats."
   - enable: true
     question: "What happens when my trials is over?"
     answer: "You will have the option to convert to a paying plan or you'll loose access to RocketSim Pro features and team build insights."
   - enable: true
     question: "Can I cancel, upgrade, or downgrade anytime?"
     answer: "Yes. You can cancel, upgrade, or downgrade your plan via your subscription settings within the web app. All cancellations and downgrades will take place at the end of your plan cycle, while upgrades will take place immediately."
+  - enable: true
+    question: "Can I download RocketSim outside of the Mac App Store?"
+    answer: "Yes, with our enterprise plan, RocketSim can be downloaded and distributed outside of the Mac App Store. **[Contact sales](mailto:ralphduin@rocketsim.app) for details.**"
```

Comment:
@ngnijland I could not get this link to show up as a white hyperlink in the styling. Can you fix that?